README.md

```diff
@@ -39,4 +39,5 @@ $ ./run.sh
 Browse to http://<your_machine_ip>
 
 **Happy Learning !!**
-
+**PRs and more than welcome**
+**If you like the project, feel free to join in for beers sometime.**
```

Comment:
@amolnaik4 Really ?

Reply:
@c0n71nu3 you need to start thinking about something other then beer also :P